### PR TITLE
Profile V3 smol adjustments

### LINKED ISSUE #9940: Profile v3 smol adjustments
looking https://deploy-preview-9931--koda-canary.netlify.app/ahp/u/15CoYMEnJhhWHvdEPXDuTBnZKXwrJzMQdcMwcHGsVx5kXYvW?redesign=true&mock=true

this could be bit smaller font and probably closer to line?
cc @exezbcz 

image

dropdown
I would put farcaster, x and web order
image

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/9931#issuecomment-2028018212_

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Design

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #9940

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)

## Screenshot 📸

Changed stats font size from 20px (`text-xl`) to 18px (`text-lg')

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

**Comment by daiagi** [comment]
@yangwao 

>and probably closer to line

what do you mean?

File: components/profile/ProfileDetailV3.vue [hunk 1]
```diff
@@ -377,27 +377,30 @@ interface ButtonConfig {
 const gridSection = GridSection.PROFILE_GALLERY
 
 const socials = {
+  [Socials.Farcaster]: {
+    icon: FarcasterIcon,
+    getUrlLabel: (value) => ({
+      label: `@${value}`,
+      url: `https://www.farcaster.xyz/${value}`,
```
Comment (yangwao) [inline components/profile/ProfileDetailV3.vue:384]:
```suggestion
      url: `https://warpcast.com/${value}`,
```

Comment (yangwao) [reply components/profile/ProfileDetailV3.vue:384]: majority is using warpcast for now

**Review by yangwao — APPROVED** [review]

**Comment by yangwao** [comment]
> what do you mean?

will discuss with exez, I mean it's first shot v3.1 so it's good like this and we will do some cosmetics

Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 9.56 USD/DOT ~ 3.138 $DOT
🧗 [1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8](https://polkadot.subscan.io/account/1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)
🔗 [0xc476d3d2c730d0cffe943e209051ac1781bb10e43a5998323843f2ab7d7b80ad](https://polkadot.subscan.io/extrinsic/0xc476d3d2c730d0cffe943e209051ac1781bb10e43a5998323843f2ab7d7b80ad)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**